dataprofiler/tests/reports/test_graphs.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import warnings
+
+from dataprofiler.profilers import IntColumn
+from dataprofiler.reports import graphs
+from unittest import mock
+import dataprofiler as dp
+import pandas as pd
+
+
+@mock.patch("dataprofiler.reports.graphs.plt.show")
+@mock.patch("dataprofiler.reports.graphs.sns")
```

Comment:
we can prob remove the sns mock since we are mocking the function which calls it: ` @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")`

Reply:
this will change the function inputs for all tests below